src/apps/Agreement/AgreementDetails.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import {
+  GU,
+  useLayout,
+  Link,
+  IdentityBadge,
+  TransactionBadge,
+} from '@aragon/ui'
+import PropTypes from 'prop-types'
+import InfoField from './InfoField'
+
+function AgreementDetails({ IPFSLink, AuthorHash, StakingHash, ContractHash }) {
```

Comment:
API nitpick: we generally use camel case instead of pascal case for the prop names.